packages/@okta/vuepress-site/guides/sign-into-spa/section03/index.md

```diff
@@ -1,16 +1,19 @@
 ---
 title: Create an Okta Application
 ---
-## Create an Okta Application
 
-In Okta, applications are OpenID Connect (OIDC) clients that can use Okta Authorization Servers to authenticate users. Your Okta org already has a default Authorization Server, so you just need to create an OIDC client that uses it.
+Before your code can redirect to Okta to sign a user in, you need to create an Okta Application that represents your single-page application.
 
-1. Sign in to the Okta Developer Console, click **Applications**, and then **Add Application**.
-2. Select **Single-Page App (SPA)** as the platform, and then click **Next**.
-3. Provide a name for your SPA application or leave the default value.
-4. Enter values for **Base URIs**. You should add the base URI of your SPA application when you develop locally. Also add any additional base URIs where your SPA application is served in production.
-5. Enter values for the **Login redirect URIs** boxes. The URI should load your SPA application at the specific route you have defined for the login redirct callback. See [Define the Login Redirect/Callback](define-the-login-redirect/callback). As with **Base URIs**, add values for local development and production.
-6. Leave **Implicit** selected for **Grant Types Allowed**.
-7. Click **Done**.
-8. On the **General** tab of the app that you just created, click **Edit** and enter the correct URI in the **Logout redirect URIs** box. <!-- See [Sign Users Out](sign-users-outlink) for more information. -->
+First, sign into the Okta Developer Console.
 
+<a href="https://login.okta.com/" target="_blank">Go to Console</a>
```

Comment:
@alexdahl-okta Should this button look the same as other navigation buttons, or would you like to see a special style?

Reply:
Perhaps a secondary button style, like the quickstart buttons on [code pages](https://developer.okta.com/code/javascript/)? `class="Button--blue"`

Will give long-term thought.